.github/workflows/build_dockers.yaml

```diff
@@ -0,0 +1,53 @@
+name: Build Dockers
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  build_docker:
+    strategy:
+      matrix:
+        env:
```

Comment:
`matrix.env` requires all combinations to be explicitly listed. Use [real matrix options](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix).

Reply:
> `matrix.env` requires all combinations to be explicitly listed. Use [real matrix options](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix).

Thanks for the heads up! Will change